server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -406,6 +396,17 @@ private void validateEmptyZKWorkerServerPaths(ServerContext context) {
     }
   }
 
+  private void initializeFateTable(ServerContext context) {
+    try {
+      Upgrader11to12.preparePre4_0NewTableState(context, SystemTables.FATE.tableId(),
+          Namespace.ACCUMULO.id(), SystemTables.FATE.tableName(), TableState.ONLINE,
+          ZooUtil.NodeExistsPolicy.FAIL);
+    } catch (InterruptedException | KeeperException ex) {
+      Thread.currentThread().interrupt();
```

Comment:
I don't think you need to re-set the interrupted flag here, because you're exiting the current thread in the next line with the ISE. I'm also wondering if this boilerplate exception handling can be pushed into the `preparePre4_0NewTableState` method, so we can just inline this method. The namespace, nodeexistspolicy, and online state can probably all be inlined.